src/dialogs/make-dialog-manager.ts

```diff
@@ -123,6 +155,42 @@ export const closeDialog = async (dialogTag: string): Promise<boolean> => {
   return true;
 };
 
+// called on back()
+export const closeLastDialog = () => {
+  if (OPEN_DIALOG_STACK.length) {
+    const lastDialog = OPEN_DIALOG_STACK.pop();
+    closeDialog(lastDialog!.dialogTag);
```

Comment:
A dialog can prevent itself from getting closed by a back action, thats why we had:

```js
  const closed = await closeDialog(dialogState.dialog);
  if (!closed) {
    // dialog could not be closed, push state again
    mainWindow.history.pushState(dialogState, "");
  }
```